guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,385 @@
+
+The flag is `true` prior to accepting, and fires `guildMemberUpdate` when the member accepts.
+
+### GuildMemberManager
+
+Methods were added to `GuildMemberManager` to provide API support for uncached members.
+
+#### GuildMemberManager#edit
+
+`guild.members.edit('123456789012345678', data, reason)`
+
+Approximately equivalent to `GuildMember#edit(data, reason)` but does not resolve to a GuildMember.
+
+#### GuildMemberManager#kick
+
+`guild.members.kick('123456789012345678', reason)`
+
+Equivalent to `GuildMember#kick(reason)`
+
+### GuildMemberRoleManager
+
+#### GuildMemberRoleManager#botRole
+
+Gets the managed role this member created when joining the guild, if any.
+
+`member.roles.botRole`
+
+#### GuildMemberRoleManager#premiumSubscriberRole
+
+Gets the premium subscriber (booster) role if present on the member.
+
+`member.roles.premiumSubscriberRole`
+
+### GuildTemplate
+
+API support for [Server Templates](https://discord.com/developers/docs/resources/template)
+
+### Integration
+
+#### Integration#roles
+
+Provides a Collection of Roles managed by the integration.
+
+### Message
+
+#### Message#crosspostable
+
+Permission helper to check if a Message can be crossposted
+
+#### Message#react
+
+Added support for `<:name:id>` and `<a:name:id>` as valid inputs to `Message#react()`
+
+#### Message#referencedMessage
+
+Gets the message this message references, if this message is a crosspost/reply/pin-add and the referenced message is cached.
+
+### MessageManager
+
+Methods were added to `MessageManager` to provide API support for uncached messages.
+
+#### MessageManager#crosspost
+
+`channel.messages.crosspost('765432109876543210')`
+
+Equivalent to `message.crosspost()`
+
+#### MessageManager#edit
+
+`channel.messages.edit('765432109876543210', content, options)`
+
+Equivalent to `message.edit(content, options)`
+
+#### MessageManager#pin
+
+`channel.messages.pin('765432109876543210', options)`
+
+Approximately equivalent to `message.pin(options)` but does not resolve to a Message.
+
+#### MessageManager#react
+
+`channel.messages.react('765432109876543210', emoji)`
+
+Approximately equivalent to `message.react(emoji)` but does not resolve to a MessageReaction.
+
+#### MessageManager#unpin
+
+`channel.messages.unpin('765432109876543210', options)`
+
+Approximately equivalent to `message.unpin(options)` but does not resolve to a Message.
+
+### NewsChannel
+
+#### NewsChannel#addFollower
+
+`channel.addFollower(channel)`
+
+Provides API support for bots to follow announcements in other channels
+
+#### NewsChannel#setType
+
+`channel.setType("text")`
+
+Allows conversion between NewsChannel and TextChannel
+
+### Role
+
+#### Role#tags
+
+Tags for roles belonging to bots, integrations, or premium subscribers.
+
+### RoleManager
+
+#### RoleManager#botRoleFor
+
+`guild.roles.botRoleFor(user)`
+
+Get the managed role a user created when joining the guild, if any.
```

Comment:
```suggestion
Gets the managed role a bot created when joining the guild, if any.
```

Reply:
"user" comes from the JSdoc comments for this method, so I don't agree with this change.

Reply:
Well, this is the method to find a bot's managed role created through the oauth flow - if it's not a bot it won't have one. Saying "user" here suggests that this might be something users can do as well